montepy/__main__.py

```diff
@@ -59,8 +65,10 @@ def main():  # pragma: no cover
     The main function
     """
     args = define_args(sys.argv[1:])
```

Comment:
Touching `sys.argv` should never be necessary when using `argparse`

Reply:
Oh really? I'll look into the preferred way.